packages/cli/commands/project/cloneApp.js

```diff
@@ -101,15 +98,26 @@ exports.handler = async options => {
     const { exportId } = await cloneApp(accountId, appId);
     const { status } = await poll(checkCloneStatus, accountId, exportId);
     if (status === 'SUCCESS') {
-      const absoluteDestPath = path.resolve(getCwd(), location);
-      const zippedProject = await downloadClonedProject(accountId, exportId);
-
-      await extractZipArchive(
-        zippedProject,
-        appName,
-        path.resolve(absoluteDestPath),
-        { includesRootDir: true, hideLogs: true }
-      );
+      // Ensure correct project folder structure exists
+      const baseDestPath = path.resolve(getCwd(), location);
+      const absoluteDestPath = path.resolve(baseDestPath, 'src', 'app');
+      fs.mkdirSync(absoluteDestPath, { recursive: true });
+
+      // Extract zipped app files and place them in correct directory
+      const zippedApp = await downloadClonedProject(accountId, exportId);
+      await extractZipArchive(zippedApp, name, path.resolve(absoluteDestPath), {
```

Comment:
Do you need `path.resolve(absoluteDestPath)` here or could you just use `absoluteDestPath `?

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1115/commits/43079c7c4627f466ab6fd739ceb2c418a72d62b1